types/react-navigation/index.d.ts

```diff
@@ -14,6 +14,7 @@
 //                 Veit Lehmann: <https://github.com/levito>
 //                 Roberto Huertas: <https://github.com/robertohuertasm>
 //                 Steven Miller <https://github.com/YourGamesBeOver>
+//                 Armando Assuncao <https://github.com/ArmandoAssuncao>
```

Comment:
Adding your name here means you'll be one of the maintainers for these typings, and you'll get notifications for issues and PRs to this definition. Just checking if you understand what it means